### PR TITLE
Fix/auth flow

### PR DESCRIPTION
**Pull request by K0sen**

### CODE REVIEW

**Comment by K0sen** [comment]
resolve

File: frontend/src/store/common/mutations.js [hunk 1]
```diff
@@ -9,3 +9,4 @@ export default {
         state.error = null;
     }
```
Comment (nemoipaha) [inline frontend/src/store/common/mutations.js]: Do we use setError, clearError ?

File: frontend/src/components/navbar/TopNavbar.vue [hunk 2]
```diff
@@ -104,18 +104,17 @@
...
-                    avatarUrl:'http://via.placeholder.com/200x200'
-                }
+export default {
+    name: 'TopNavbar',
+    computed: mapGetters({
+        isUserLoggedIn: 'hasToken'
+    }),
+    data() {
+        return {
+            user: {
```
Comment (nemoipaha) [inline frontend/src/components/navbar/TopNavbar.vue]: Sync with dev, user profile is implemented in another way

File: config/jwt.php [hunk 3]
```diff
@@ -100,7 +100,7 @@
     |
     */
 
-    'ttl' => env('JWT_TTL', 60),
+    'ttl' => env('JWT_TTL', 1),
```
Comment (nemoipaha) [inline config/jwt.php]: This value in minutes?

**Comment by K0sen** [comment]
returned it back

**Pull request merged**